examples/10-break-a-cipher.mjs

```
@@ -0,0 +1,111 @@
+        const shift = key.toUpperCase().charCodeAt(at++ % key.length) - 65;
+        const out = String.fromCharCode(65 + (upper.charCodeAt(0) - 65 + shift) % 26);
+        return ch === upper ? out : out.toLowerCase();
+    }).join("");
+}
+
+const client = await connect("break-a-cipher");
+try {
+    step("Is it plaintext already? The cheapest question, so it goes first");
+    const already = await call(client, "cyberchef_plaintext_check", { input: PLAIN });
+    const alreadyParsed = JSON.parse(already);
+    console.log(`  verdict "${alreadyParsed.verdict}" via ${alreadyParsed.decided_by}`);
```

ESLint

[error] Unexpected console statement.

(no-console)